docs/DeveloperGuide.md

```diff
@@ -378,7 +387,7 @@ rather insignificant as once the user gets acquainted with the command, reading
 In GuestBook, the user can find guests using the `find` command. The extensibility of this `find` feature makes it easy for the user to locate guests in GuestBook.
 
 <span style="font-size: large; color: #e46c0a">Implementation:</span>
-* The `find` command takes in multiple keywords separated by spaces, and find all guests whose `fields` contain any of the keywords. The keywords are case-insensitive as well. For example, finding 'Alice' is the same as finding 'aLiCE'.
+* The `find` command takes in multiple keywords separated by spaces, and find all guests whose fields contain any of the keywords. The keywords are case-insensitive as well. For example, finding 'Alice' is the same as finding 'aLiCE'.
```

Comment:
Should we use "search terms" rather than "keywords" to make it more in sync with the explanation below?

Reply:
Agreed!